libsrc/cx16/tgi/cx640p1.s

```diff
@@ -0,0 +1,677 @@
+;
+; Graphics driver for the 640 pixels across, 480 pixels down, 2 color mode
+; on the Commander X16
+;
+; 2024-06-11, Scott Hutter
+; Based on code by Greg King
+;
+
+        .include        "zeropage.inc"
+
+        .include        "tgi-kernel.inc"
+        .include        "tgi-error.inc"
+
+        .include        "cbm_kernal.inc"
+        .include        "cx16.inc"
+
+        .macpack        generic
```

Comment:
Is this actually needed?

Reply:
Apparently so.  I believe (not sure really) that module_header requires this.  Removing the ,macpack lines caused errors in compilation that admittedly I dont fully understand.  But it the macpack lines exist in the Commodore tgi drivers as well.